http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -317,6 +318,14 @@ class ProxyRoutesSpec
     }
   }
 
+  it should "add Galaxy remote user headers" in {
+    Get(s"/proxy/google/v1/apps/$googleProject/$appName/$serviceName")
+      .addHeader(Cookie(tokenCookie)) ~> httpRoutes.route ~> check {
+      responseAs[Data].headers.toList should contain allElementsOf Map("HTTP_REMOTE_USER" -> "galaxy-user",
```

Comment:
so we're setting user and secret in response and galaxy UI knows how to read this from response header?

Reply:
Yeah, we set it in the request header to Galaxy and the secret is also configured on the Galaxy side:

https://github.com/galaxyproject/galaxy/blob/d6ba85ae48072bc9c15f17955588ef1576dc5393/lib/galaxy/config/sample/galaxy.yml.sample#L1397